### PR TITLE
Liza/voting on question 2

### PR DESCRIPTION
**Pull request by lizamcpherson**
If you access the individual question from the qset page and quiz yourself, you can now vote on the quality of the question from that page after you have submitted your answer. You can see the vote count change immediately, and the updated count will show up on the qset page when you navigate back there as well.

### CODE REVIEW

File: app/views/questions/show.html.erb [hunk 1]
```diff
@@ -19,12 +19,17 @@
   <div class="prettyform answers" style="display:none">
     <div class="box center">
       <h3>Answer:</h3>
+      <div class="net-votes"><p id="question-<%= @question.id %>-net-votes"><%= @question.plusminus %></p><br></div>
+            <div class="question-voting">
```
Comment (katloboda) [inline app/views/questions/show.html.erb]: There shouldn't be this indentation.

File: app/views/questions/show.html.erb [hunk 2]
```diff
@@ -19,12 +19,17 @@
...
+              <div class="glyphicon glyphicon-triangle-top upvote-button" data-vote="upvote" data-vote-qid="<%= @question.id %>"></div>
+              <div class="glyphicon glyphicon-triangle-bottom downvote-button" data-vote="downvote" data-vote-qid="<%= @question.id %>"></div>
+            </div>
       <div class="correct-container">
         <h4 class="correct"><%= @question.answers[0].text %></h4>
       </div>
     </div>  
-  </div>
-  <div class="prettyform response" style="display:none">    
+  </div>
```
Comment (katloboda) [inline app/views/questions/show.html.erb]: There shouldn't be spacing after the div,or after any content

File: config/secrets.yml [hunk 3]
```diff
@@ -17,4 +17,4 @@ test:
   secret_key_base: ee4280f086c62260dbd973a322a84720db40913555b2ee165b04b02a0ba4907b39d74d0819f15913d8c6ce1b80c91be12f181322474ea150a9d425619db0101a
 
 production:
-  secret_key_base: <%= ENV["askup_secret_key_base"] %>
```
Comment (katloboda) [inline config/secrets.yml:20]: The secret_key_base should not be hardcoded for the production version of the site. Just change it back to what it used to be.

File: config/secrets.yml [hunk 4]
```diff
@@ -17,4 +17,4 @@ test:
...
+  secret_key_base: 29707b42eaa65ec7e43f5ba16d7c2f73656bb7bee22c7cc28bb08fd819a46dde3ce01700035d196d3d74a5f16162f72478d3dfe1284e127d8312f2f1d3a447bf
```
Comment (elliottyates) [inline config/secrets.yml]: Please note @katloboda's comment re: `secret_key_base`.

**Comment by elliottyates** [comment]
Nice, it works well. I'm inclined to get the functionality in there, then loop back and get feedback from the PO on the appearance/design -- we might want to move it around on the page or something. Iterate, iterate, iterate.

When the secrets.yml change has been rolled back this can be merged.

**Comment by elliottyates** [comment]
Closing -- looks like this is superseded by #83.

**Pull request closed without merging**